app/controllers/api/v2/geographical_areas_controller.rb

```diff
@@ -25,6 +25,10 @@ def serialized_geographical_area
       def geographical_area
         GeographicalArea.actual.by_id(params[:id]).eager(:geographical_area_descriptions, :contained_geographical_areas).take
       end
+
+      def exclude_none
+        params.fetch(:filter, {}).permit(:exclude_none).fetch(:exclude_none, false) == 'true'
```

Comment:
thinking about this:
if `exclude_none` is not present,
it returns `false` which is a `Boolean`,
and it is compared to `true` which is a `String`
... which sounds weird 🤔

Reply:
I think what's weird is we shouldn't have a separate endpoint for countries, it should also just be a filter

Reply:
`/api/v2/geographical_areas?filter['countries']=false,filter['exclude_none']=true`